app/models/subscription.rb

```diff
@@ -9,6 +9,7 @@ class Subscription < ApplicationRecord
   enum frequency: FREQUENCY_OPTIONS
 
   has_many :alert_runs
+  has_many :job_alert_feedbacks
```

Comment:
Can you leave more than one feedback for a subscription?

Reply:
Yes. You can leave positive feedback. Then maybe 4 months later you decide to leave negative feedback. Maybe you edited the alert in between.